src/video/kmsdrm/SDL_kmsdrmvulkan.c

```diff
@@ -384,7 +384,13 @@ SDL_bool KMSDRM_Vulkan_CreateSurface(_THIS,
         new_mode_parameters.visibleRegion.height = window->h;
         /* SDL (and DRM, if we look at drmModeModeInfo vrefresh) uses plain integer Hz for
            display mode refresh rate, but Vulkan expects higher precision. */
-        new_mode_parameters.refreshRate = window->fullscreen_mode.refresh_rate * 1000;
+        {
+            double rr = 0.0;
+            if (window->fullscreen_mode.refresh_rate_denominator) {
```

Comment:
This should be `if (window->fullscreen_mode.refresh_rate_numerator)`

Reply:
it think it better to check denominator to avoid division by 0.

and if, numerator is 0, it will gives 0 anyway